### PR TITLE
Enable New Relic and Gunicorn with CKAN

### PR DESCRIPTION
**Pull request by robert-bryson**
related to https://github.com/GSA/datagov-deploy/issues/3569

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -21,7 +21,7 @@ DB_CKAN_DB=ckan
 
 # Put envvars last per envvars doc, https://github.com/okfn/ckanext-envvars#requirements
```
Comment (mogul) [inline .env:22]: Should we update this comment...?

File: config/ckan.ini [hunk 2]
```diff
@@ -130,7 +130,7 @@ ckan.auth.public_user_details = false
 #		same origin policy
 
 # Put envvars last per envvars doc, https://github.com/okfn/ckanext-envvars#requirements
```
Comment (mogul) [inline config/ckan.ini:132]: Same here

File: .env (hunk 1 above)
Comment (jbrown-xentity) [reply .env:22]: @robert-bryson that's my bad, I should have had you put that back after we tried flipping that around... I would put envvars last again.

**Review by jbrown-xentity — DISMISSED** [review]

File: .env [hunk 3]
```diff
@@ -98,7 +98,6 @@ CKANEXT___GOOGLEANALYTICS__IDS=google-analytics-fake-key-testing-87654321
 # The maximum content size, in bytes, for uploads
 CKAN__STORAGE__MAX_CONTENT_LENGTH=650000000
 CKAN_STORAGE_PATH=/var/lib/ckan/files
-CKANEXT__S3FILESTORE__AWS_BUCKET_NAME
```
Comment (nickumia-reisys) [inline .env:101]: Why was this line deleted?

**Review by nickumia-reisys — CHANGES_REQUESTED** [review]

File: .env (hunk 3 above)
Comment (robert-bryson) [reply .env:101]: It was throwing an error for me during my local testing. The full string is the line below: `CKANEXT__S3FILESTORE__AWS_BUCKET_NAME=inventory`. I'm not sure why it wasn't causing an issue in other cases, to be honest.

Comment (nickumia-reisys) [reply .env:101]: Interesting.. okay.  Yeah, that makes sense 👍

**Review by nickumia-reisys — APPROVED** [review]
